README.md

```diff
@@ -184,7 +184,7 @@ See [CONTRIBUTING](CONTRIBUTING.md) for details on submitting patches and the co
 
 ## Reporting bugs
 
-See [reporting bugs](Documentation/reporting_bugs.md) for details about reporting any issues.
+See [reporting bugs](https://etcd.io/docs/latest/reporting-bugs) for details about reporting any issues.
```

Comment:
This is a broken link. However, https://etcd.io/docs/v3.4.0/reporting_bugs/ works.